azkaban-common/src/main/java/azkaban/server/HttpRequestUtils.java

```diff
@@ -115,11 +116,32 @@ public static ExecutionOptions parseFlowOptions(final HttpServletRequest req,
       execOptions.setSlaOptions(slaOptions);
     }
 
+    final Map<String, Map<String, String>> runtimePropertiesGroup = getMapParamGroup(req,
+        "runtimeProperty");
```

Comment:
The form is singular because of how these are passed as request params.

A single request param is for example `runtimeProperty[some:node][foo]` and it targets just a single property `foo`.

So the request may include:
```
runtimeProperty[some:node][prop1] = val1
runtimeProperty[some:node][prop2] = val2
..
```

It follows the same pattern as the existing `flowOverride`.

I think it would be a bit off to use plural name, because then the request would be like:
```
runtimeProperties[some:node][prop1] = val1
runtimeProperties[some:node][prop2] = val2
..
```

The response format is different. It has a single param `runtimeProperties` with nested content.